Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -5410,10 +5410,26 @@ void MainWindow::slotFilesContextMenu(const QPoint &/* pos*/)
 {
     QMenu menu(this);
     QMenu* sortMenu = menu.addMenu(tr("Sort By..."));
-    QAction* sortName = sortMenu->addAction(tr("&Name"));
-    QAction* sortSize = sortMenu->addAction(tr("&Size"));
-    QAction* sortOwner = sortMenu->addAction(tr("&Owner"));
-    QAction* sortUpload = sortMenu->addAction(tr("&Upload Date"));
+    QAction* sortName = new QAction();

```

Comment:
To avoid memory leak "sortMenu" should be parent of QAction: "new QAction(sortMenu)"